<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Format_Dialog</class>
 <widget class="QDialog" name="Format_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_rom_size">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>rom_size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_system">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>system</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_system">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ntfs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>vfat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ext4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_device_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>device_name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_device_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>291</width>
     <height>26</height>
    </rect>
</property>
    <property name="maxLength">
    <number>11</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_clean_or_not">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>clean it total</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rom_size">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_process">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_process">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
